add compatibility with disable flag

From: ivanekovtune-at-githedgehog <ivan@githedgehog.com>

remove bgpd_bmp.so from frr.install to make this file compatible with disable flag
---
 debian/frr.install |    1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/frr.install b/debian/frr.install
index 48263222f..60803540f 100644
--- a/debian/frr.install
+++ b/debian/frr.install
@@ -7,7 +7,6 @@ usr/bin/vtysh
 usr/lib/*/frr/libfrr.*
 usr/lib/*/frr/libfrrcares.*
 usr/lib/*/frr/libfrrospfapiclient.*
-usr/lib/*/frr/modules/bgpd_bmp.so
 usr/lib/*/frr/modules/dplane_fpm_nl.so
 usr/lib/*/frr/modules/zebra_cumulus_mlag.so
 usr/lib/*/frr/modules/zebra_fpm.so
